scripts/init

```diff
@@ -28,6 +34,8 @@ devenv() {
 
     display "'${DEVENVFLAVOR}' to be turned off"
     devenv_act nameremove ${DEVENVFLAVOR}
+  elif [[ "$1" == "help" ]]; then
```

Comment:
This is needed for printing help message.

Reply:
After the second commit 882c52f4d3b2437bfa2609f3ca9f703164af86e1 this new `devenv help` is no longer required, but still good to have.